lib/iris/tests/test_analysis.py

```diff
@@ -1192,8 +1192,6 @@ def test_rotate_1d(self):
 
 @_shared_utils.skip_data
 class TestAreaWeights:
-    # Note: chunks is simply ignored for non-lazy data
-    @pytest.mark.parametrize("chunks", [None, (2, 3)])
```

Comment:
`chunks` is not used here, and [marks cannot be applied to fixtures anymore](https://docs.pytest.org/en/stable/deprecations.html#applying-a-mark-to-a-fixture-function), so this can be safely removed.